source/strings.tex

```diff
@@ -1653,12 +1653,14 @@
 by causing reallocation.
 
 \pnum
-\complexity Linear in the size of the sequence.
+\complexity If the size is not equal to the old capacity,
+linear in the size of the sequence;
+otherwise constant.
 
 \pnum
 \remarks Reallocation invalidates all the references, pointers, and iterators
 referring to the elements in the sequence as well as the past-the-end iterator.
```

Comment:
There's a missing edit here to add a comma.

Reply:
The remaining edits from the issue resolution (changes to deque, string) were also missing! Fixed.

Reply:
@zygoloid: I said that in my comment above!